src/Symfony/Component/HttpFoundation/Tests/RequestTest.php

```diff
@@ -929,6 +929,24 @@ public function testGetContentReturnsResource()
      */
     public function testGetContentCantBeCalledTwiceWithResources($first, $second)
     {
+        if (PHP_VERSION_ID >= 50600) {
+            $this->markTestSkipped('PHP >= 5.6 allows to open php://input several times.');
+        }
+
+        $req = new Request();
+        $req->getContent($first);
+        $req->getContent($second);
+    }
+
+    /**
+     * @dataProvider getContentCantBeCalledTwiceWithResourcesProvider
+     */
+    public function testGetContentCanBeCalledTwiceWithResources($first, $second)
+    {
+        if (PHP_VERSION_ID < 50600) {
+            $this->markTestSkipped('PHP < 5.6 does not allow to open php://input several times.');
+        }
+
         $req = new Request();
         $req->getContent($first);
         $req->getContent($second);
```

Comment:
On 5.3 if you remove the `markTestSkipped()` call it will fail because a `\LogicException` is thrown.
The only thing that this test do is checking that we don't throw a `\LogicException` when it's not necessary (on PHP 5.6+).

I try explain me better:
if you run that test on PHP 5.6 without the patch, this test will fail because a `\LogicException` will be thrown at the second call.